packages/shared/src/utils/screen/responsive.ts

```diff
@@ -1,3 +1,13 @@
+declare global {
+    interface Navigator {
+        msMaxTouchPoints: number;
+    }
```

Comment:
@niloo-fs I guess it should be `maxTouchPoints` instead and no need to define it 🤔

Reply:
@farzin-fs why no need to define it? :thinking:

Reply:
Because it's in global this type and we don't need to define types for it

Reply:
@farzin-fs let's don't change the code! maybe we break the code somewhere :))

Reply:
@niloo-fs Yeah let’s go with what we have and work 😅🙇🏻